forgerock-auth/src/main/java/org/forgerock/android/auth/RequestInterceptorFactory.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2020 ForgeRock. All rights reserved.
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+package org.forgerock.android.auth;
+
+import lombok.Getter;
+
+/**
+ * Factory to manage {@link RequestInterceptor}
+ */
+public class RequestInterceptorFactory {
```

Comment:
Should this factory be on 'forgerock-core'?

Reply:
Moved to core